### ISSUE TITLE
Date and place

### ISSUE THREAD

**Issue by webmaxru** [webmaxru/progressive-web-conf#1]

**Comment by webmaxru** [comment]
I propose:

**Date**
Late October 2018.

*Reasoning*
Autumn 2018 is the perfect moment for the gathering:
- foundations of PWA (Service Worker API) is in production in all major browser engines
- the level of interest from the developers' side is huge, but there is a serious lack of information about the roadmaps from browser/platform vendors
- we can't do it later because we'll lose the momentum
- we can't do it earlier because we need some time for the organizing

**Location**
London

*Reasoning*
Many folks from browsers devrel teams are located there. At least from Google, Mozilla, Samsung Internet. This will be easier to organize the logistics.

**Organizing**
We need a team of core organizers and contributors. I propose to have the team as volunteers (and I nominate myself for being a part of the team) and hire some event agency to help us with the basic tasks. Of course, we'll need some sponsors and an organization (registered as not-for-profit) to operate on behalf of.

We could hold all the public discussions at this repo and all the private ones on PWA slack in the #pwaconf private group (it's there for a long time).

**Comment by comp615** [comment]
As mentioned in #3, I think getting the browser's perspectives on how a long-term dev-browser relationship might look could inform this. For instance, Chrome may have PR constraints and not be able to share their plans before Chrome Dev Summit. If so, then we could not productively have an honest conversation about future roadmaps; maybe it should occurs after that? Or maybe they'd be open to sharing before in a small group setting. Worth checking. (Maybe you can tag them on #3)

To location, I think SF is a strong choice as most of the vendors are on the west-coast US and if we want to rally around them, making it easy might be helpful...again just depends on the focus of the event.

**Comment by webmaxru** [comment]
Thank you for your input, @comp615!

My thought is: if we go after CDS - we'll only repeat the info from there. It's better to bring at least tiny piece of new information to the developers + Chrome team is good with sharing their plans + Chrome is only one of the many browser vendors we want to hear from. This is why I don't see an issue to run before CDS.

About location - as I mentioned, I know that devrels and PWA-speakers from Google (Paul Kinlan, Jake Archibald), Mozilla (Andreas Bovens), Samsung Internet (Ada Rose Cannon, Daniel Appelquist, Peter O'Shaughnessy), Microsoft (Simona Cotin) are based in London - this is why exactly this city came up to my mind :)

Anyway, both date and location to be discussed.

**Comment by poshaughnessy** [comment]
Just a quick note that 25th October would clash with [TPAC in Lyon](https://www.w3.org/wiki/TPAC/2018), which I presume @torgo and various other browser folks would be planning to attend.

**Comment by gavmck** [comment]
Hey there, @webmaxru asked me to come introduce myself here over in the PWA Slack.

I'm Gav, a front-end developer working at [Etch](https://etch.co). 
We are a software development agency focused on PWAs and [making better software](https://makebetter.software/).

(I|We)'d love to help out with the PWA conf in some manner, in any avenue from talking (with such previous hits as "[I have an idea for an app](https://github.com/gavmck/i-have-an-idea-for-an-app)" at UKC App conf), to helping with the promo site, to just being on hand giving out lanyards.

**Comment by docluv** [comment]
You are correct October has lots of developer events. I have 2-4 on my calendar already :)

**Comment by webmaxru** [comment]
@poshaughnessy @torgo Me and Diego discussed a crazy idea about the date/location: why not to hold it in Lyon during TPAC (or a day before, or a day after)?
- Browser folks will be there
- Still okay logistics for dev community speakers
- Hopefully, w3c could help with the organizing?

What do you think?

**Comment by Sneiwert** [comment]
If we choose to have it in London it would be easier for us who already organise the[ London PWA Meetups](https://www.meetup.com/London-Progressive-Web-Apps/) to organise the event together. As long as the conference is non-profit we can use [Skills Matter](https://skillsmatter.com/event-space#event_spaces) as a venue for free. Then they'll also help us with organising.

**Comment by webmaxru** [comment]
2019 is coming, so let's reschedule the event. What about March?

**Comment by christianliebel** [comment]
> 2019 is coming, so let's reschedule the event. What about March?

The MVP Summit takes place from March 17–22. Apart from that, March looks good to me.

**Comment by Sneiwert** [comment]
End of March, the last week would be great. I'll contact Skills Matter
again and check if they could sponsor it, or if Enonic can pay for the
conference rooms we would need.

Siw
